source/quickstart.txt

```diff
@@ -1,12 +1,235 @@
+         is in the ``bin`` directory. 
+
+      For operating system specific installation instructions, see
+      :ref:`c2c-install`.
+
+   .. step:: Connect the Clusters
+
+      A connection string contains the network and authentication
+      details that ``mongosync`` needs to connect to the source and
+      destination clusters.
+
+      Determine the IP address and port for your source and destination
+      clusters. You will use this information and the user
+      authentication details to construct the connection strings. 
+
+      The standard URI connection scheme has the form:
+
+      .. code-block:: shell
+
+         mongodb://[username:password@]host1[:port1][,...hostN[:portN]][/[defaultauthdb][?options]]
+
+      Your connections strings will resemble:
+
+      .. code-block:: shell
+
+         cluster0:
+         mongodb://clusterAdmin:superSecret@clusterOne01.fancyCorp.com:20020,clusterOne02.fancyCorp.com:20020,clusterOne03.fancyCorp.com:20020
+         cluster1:
+         mongodb://clusterAdmin:superSecret@clusterTwo01.fancyCorp.com:20020,clusterTwo02.fancyCorp.com:20020,clusterTwo03.fancyCorp.com:20020
+
+      For more details, see :ref:`c2c-connecting`.
+
+Initialize ``monogsync``
+------------------------
+
+``monogsync`` must create an initial connection to the source and
+destination clusters before it can start to sync data. To create the
+initial connection, issue the following command on a single line (the
+command is reformated here for clarity):
+
+.. code-block:: shell
+
+   ./bin/mongosync --cluster0 mongodb://clusterAdmin:superSecret@clusterOne01.fancyCorp.com:20020,
+                              clusterOne02.fancyCorp.com:20020,
+                              clusterOne03.fancyCorp.com:20020
+                   --cluster1 mongodb://clusterAdmin:superSecret@clusterTwo01.fancyCorp.com:20020,
+                              clusterTwo02.fancyCorp.com:20020,
+                              clusterTwo03.fancyCorp.com:20020
+
+Initialization Notes
+~~~~~~~~~~~~~~~~~~~~
+
+- When ``mongosync`` first connects to the source and destination
+  clusters it is in the :ref:`IDLE <c2c-states-descriptions>` state.
+- ``mongosync`` does not synchronize data until it receives the
+  :ref:`start <c2c-api-start>` command. 
+- Designate the source and destination clusters with the :ref:`start
+  <c2c-api-start>` command. "cluster0" and "cluster1" are just labels,
+  either cluster can be ``cluster0`` or ``cluster1``.
+
+Synchronize Data Between Clusters
+---------------------------------
+
+The command interface for ``mongosync`` is an HTTP server that
+publishes a REST API. To control ``mongosync``, use the API endpoints.
+
+The :ref:`start <c2c-api-start>` command initiates data synchronization.
+To start synchronizing, use ``curl`` or a similar program to issue the
+:ref:`start <c2c-api-start>` command:
+
+.. literalinclude:: /includes/api/requests/start.sh
```

Comment:
Same issue here with copying into a terminal. Either should be 1 line, or add a `\` at the end of the first line.